src/cli/lint/autofix.ts

```diff
@@ -79,6 +81,17 @@ const shouldPush = async ({
     return false;
   }
 
+  if (currentBranch?.startsWith(RENOVATE_DEFAULT_PREFIX)) {
+    try {
+      await GitHub.getPullRequestNumber();
+    } catch (error) {
+      log.warn(
+        'Could not find an open pull request for this branch. The autofix commit will not be applied to allow Renovate to open a pull request successfully.',
+      );
```

Comment:
I'll review this proper in a bit but seems perfectly reasonable. Could we also write a Buildkite warning and share next steps (i.e. fix is available but we did not push it, if the PR has since been created then you can manually rerun the step to push).